pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -66,6 +68,7 @@ func NewReconciler(scope *actuators.MachineScope) *Reconciler {
 		networkInterfacesSvc:  networkinterfaces.NewService(scope.Scope),
 		virtualMachinesSvc:    virtualmachines.NewService(scope.Scope),
 		virtualMachinesExtSvc: virtualmachineextensions.NewService(scope.Scope),
+		disksSvc: disks.NewService(scope.Scope),
```

Comment:
may not be formatted.

Reply:
fixed, dunno why `fmt` didn't hit this automatically on file save.